react/features/participants-pane/components/native/ParticipantsPane.js

```diff
@@ -1,19 +1,19 @@
 // @flow
 
+import { useNavigation } from '@react-navigation/native';
 import React, { useCallback } from 'react';
 import { useTranslation } from 'react-i18next';
-import { View } from 'react-native';
+import { TouchableWithoutFeedback, View } from 'react-native';
 import { Button } from 'react-native-paper';
 import { useDispatch, useSelector } from 'react-redux';
 
 import { openDialog } from '../../../base/dialog';
-import { JitsiModal } from '../../../base/modal';
+import { Icon, IconClose } from '../../../base/icons';
 import {
     isLocalParticipantModerator
 } from '../../../base/participants';
 import MuteEveryoneDialog
     from '../../../video-menu/components/native/MuteEveryoneDialog';
-import { close } from '../../actions.native';
```

Comment:
Can we delete this now?

Reply:
Yes because now when we close the screen we navigate() back to conference